client/src/components/quiz/api/Lab4/quizQuestions.js

```diff
@@ -75,11 +87,13 @@ const quizQuestionsLab4 = [
     answers: [
       {
         val: 1,
-        content: "True",
+        type: "0",
+        content: "Yes",
```

Comment:
if this is yes and no wouldn't it make sense to change these to be just booleans?

Reply:
booleans make sense, was an over sight when fixing to work for new component